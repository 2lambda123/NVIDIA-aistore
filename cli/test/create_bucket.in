ais create bucket $BUCKET_1
ais create bucket $BUCKET_1
ais rm bucket $BUCKET_1
