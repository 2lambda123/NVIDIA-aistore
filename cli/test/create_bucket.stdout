$BUCKET_1 bucket created
Bucket "$BUCKET_1" already exists
$BUCKET_1 bucket destroyed
